pydantic_numpy/helper/annotation.py

```diff
@@ -75,7 +128,7 @@ def __get_pydantic_core_schema__(
             python_schema=core_schema.chain_schema([_common_numpy_array_validator, np_array_schema]),
             json_schema=np_array_schema,
             serialization=core_schema.plain_serializer_function_ser_schema(
-                _serialize_numpy_array_to_data_dict, when_used="json"
+                cls.serialize_numpy_array_to_json, when_used="json"
```

Comment:
`serialize_numpy_array_to_json` is an instance variable, and [not a class variable](https://github.com/caniko/pydantic-numpy/pull/32/files#diff-35ff13acb747a8fc01e3dbee84763b158eba7f8b0d7d43741e626f41ba734a32R78). You can fix this by wrapping the typing in `typing.ClassVar`.

Reply:
No :man_shrugging: